app/models/vacancy.rb

```diff
@@ -2,6 +2,9 @@
 
 # rubocop:disable Metrics/ClassLength
 class Vacancy < ApplicationRecord
+  # needed to remove legacy columns
+  self.ignored_columns += %w[safeguarding_information_provided safeguarding_information]
+
```

Comment:
This belongs in the other PR, along with associated analytics changes. The only thing here should be the migration IMHO